### DISCUSSION TITLE
Add Pinch-to-Zoom Functionality

### DISCUSSION THREAD

**Discussion by tassosgeo** [Foblex/f-flow#127] [Feature Request] [answered]
### Description

It could be good if pinch-to-zoom functionality is added.
This feature would allow users to zoom in and out using a pinch gesture, which is useful for mobile, tablet devices and in touch screens in general.

**Comment by siarheihuzarevich** [comment, marked as answer]
Hi! Great suggestion — pinch-to-zoom support would definitely improve usability on mobile and touchscreen devices.

This feature is already on the roadmap, and we’re planning to add gesture-based zooming in an upcoming release.
Thanks for the feedback and your interest in the library!

**Comment by siarheihuzarevich** [comment]
Hey! Thanks for the question 🙌

Yes - this is supported in **Foblex Flow**.

You can zoom the canvas in a few different ways:
- **Mouse wheel**
- **Double click**
- **Buttons**
- And also **pinch-to-zoom** (trackpads / touch devices)

To enable zooming, just add the `fZoom` directive to your `<f-canvas>`.

✅ Live example: https://flow.foblex.com/examples/zoom

If anything is unclear in your setup - share a small snippet and I’ll point you to the right place.

And if you like what we’re building, please consider starring the repo ⭐️ It helps a lot!
